src/main/resources/templates/fragments/piwik.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html xmlns:th="http://www.thymeleaf.org">
+<head>
+  <title>Piwik</title>
+</head>
+<body>
+<div th:fragment="piwik">
+  <script th:inline="javascript">
+                (function() {
+                    bindPiwikListener('web', /*[[${@environment.getProperty('piwik.url')}]]*/, /*[[${@environment.getProperty('piwik.siteId')}]]*/);
```

Comment:
Can the first argument here be changed to 'company-accounts' for the clarity in the Piwik dashboard?

Reply:
Perfect, I will do. I wasn't sure what the argument i had to used